### PR TITLE
Add asset commands

### PR DESCRIPTION
**Pull request by sergey-borovkov**
## What does this PR do?
Adds asset-related commands.
Upload progress bar:
image

## GitHub issue or Phabricator ticket number?
PH T6405
## How has this been tested?
Manually + unit tests
## Checklist before merging

- [x] If have added tests.
- [x] Is this a new feature? If yes, please write one phrase about this update.
- [x] I've attached relevant screenshots (if relevant).

### CODE REVIEW

File: src/commands.rs [hunk 1]
```diff
@@ -101,36 +111,97 @@ impl Formatter for Screens {
     }
 }
 
+impl Assets {
+    pub fn new(value: serde_json::Value) -> Self {
+        Self { value }
```
Comment (korvyashka) [inline src/commands.rs:116]: Isn't it what default constructor does?:
```
let asset: Assets = Assets{value: value};
```

**Review by korvyashka — APPROVED** [review]

**Comment by sergey-borovkov** [comment]
Does not matter ? What if we have more stuff to do here later?

File: src/commands.rs (hunk 1 above)
Comment (renatgalimov) [reply src/commands.rs:116]: No. Constructing with `Assets {value: value}` requires you to know about the object structure while when constructing from new all the magic happens inside new. Encapsulation.

**Review by renatgalimov — APPROVED** [review]

File: src/commands.rs [hunk 2]
```diff
@@ -163,14 +234,66 @@ impl ScreenCommand {
...
+    pub fn add(&self, path: String, title: String) -> anyhow::Result<Assets, CommandError> {
+        let url = format!("{}/v4/assets", self.authentication.config.url.clone());
+
+        let mut headers = HeaderMap::new();
+        headers.insert("Prefer", "return=representation".parse()?);
+
+        let file = File::open(path)?;
+        let file_size = file.metadata()?.len();
+        let pb = ProgressBar::new(file_size);
+        println!("Uploading asset.");
```
[the remark this answers is not in the document]
Comment (sergey-borovkov) [reply src/commands.rs:266]: Will leave that for later.

**Pull request merged**